app/controllers/unattended_controller.rb

```diff
@@ -171,7 +168,15 @@ def verify_found_host
     end
 
     error = host_verifier.errors.first
-    render_error(error[:type], error[:message], error[:params])
+    if ipxe_request?
+      render_ipxe_message(error[:message])
```

Comment:
I believe this needs to be

```suggestion
      render_ipxe_message(message: error[:message])
```

But shouldn't `render_error` handle this on it's own?